src/services/products.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+import { ProductResponse } from 'components/pages/Products/Products.types'
+
+const productsAPI = axios.create({
+  baseURL: import.meta.env.VITE_BASE_URL,
+})
```

Comment:
the instance should be contained in your project's config directory as all of our services will use it :D